langchain/langchain.config.js

```diff
@@ -65,6 +65,8 @@ export const config = {
     "chains/retrieval": "chains/retrieval",
     "chains/sql_db": "chains/sql_db/index",
     "chains/graph_qa/cypher": "chains/graph_qa/cypher",
+    // chat models
+    "chat_models": "chat_models/configurable",
```

Comment:
I'm ok with `chat_models/configurable` as the entrypoint but remember to keep naming here constant

I've started relying on it for some things in the build script

Reply:
Also fine with `chat_models/universal` but check w/ Bagatur about serialization

Reply:
Oh wait - actually yeah let's do `chat_models/universal` or `configurable`. We need to be very careful that this is in its own entrypoint with nothing else since it's Node only.

Reply:
Okay can update, this deviates from PY but ig it's not that big an issue

Reply:
Yeah I think we have to in this case since we should reserve the stock `chat_models` entrypoint for something more generic